notebooks/05a Modelling - regression.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Modelling - Regression\n",
+    "\n",
+    "This notebook explores regression models to **predict Length of Stay (days)** as a baseline to the [Long Stayer Risk Stratification](https://github.com/nhsx/skunkworks-long-stayer-risk-stratification) model which achieved a Mean Absolute Error **(MAE) of 3.8 days** (2.2 median absolute error).\n",
```

Comment:
Adding an overview of the models used in the notebook and why will be be helpful.

I would also consider breaking down the models in to separate notebooks.